app/controllers/api/v1/widget/contacts_controller.rb

```diff
@@ -11,6 +11,13 @@ def update
     render json: contact_identify_action.perform
   end
 
+  # TODO : clean up this with proper routes delete contacts/custom_attributes
+  def destroy_custom_attributes
+    @contact.custom_attributes = @contact.custom_attributes.excluding(params[:custom_attributes])
```

Comment:
What is being sent here? Is it an array of strings or just string?

Reply:
We can pass array of strings